pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -632,17 +646,23 @@ def _distribution_new_customers(
             s = pm.HalfFlat("s")
             beta = pm.HalfFlat("beta")
 
-            # This is the shape if using fit_method="map"
-            if self.fit_result.dims == {"chain": 1, "draw": 1}:
+            if shape_kwargs is None:
```

Comment:
Admittedly, I rewrote this because that IF statement was preventing 100% testing coverage. I don't see much use for it other than adjusting the `shape` parameter.

Reply:
I could use some help to fix this: the mock idata object for amcmc fit isn't triggering the IF statement in testing, so it has the same dims as a MAP fit. It seems to work fine when tested in the notebook though.

Also, distribution_customer_population has an additional dimension due to the T parameter. I've added an argument to make this adjustable, but it may be more trouble than it's worth.

Reply:
What are the dims of the fake map fit? You should have chains=1, draws=1. Perhaps you have too put one extra bracket around the parameters

Reply:
Found the issue - the idata object wasn't resetting between the `mcmc` and `map` testing parametrizations.

I had to increase the `rtol` to get the tests to pass. This makes sense for the `mcmc` fits because there are only 100 samples, but I'm not sure why it also had to be increased for `map` since nothing otherwise was changed except my dev env's version of `numpy`.